tests/t590-operator.c

```diff
@@ -84,6 +85,9 @@ int main(int argc, char **argv) {
   CeedOperatorSetField(op_mass, "v", elem_restriction_u, basis_u, CEED_VECTOR_ACTIVE);
   CeedOperatorAtPointsSetPoints(op_mass, elem_restriction_x_points, x_points);
 
+  CeedOperatorIsAtPoints(op_mass, &is_at_points);
+  if (!is_at_points) printf("Error: Operator should be at points\n");
```

Comment:
Minor - we should replace any use of `op->is_at_points` with this helper

Reply:
Alright, just updated